src/Compilers/CSharp/Test/Syntax/Parsing/LambdaAttributeParsingTests.cs

```diff
@@ -1207,34 +1187,24 @@ public void PrefixOperator()
             UsingExpression("-- [A] () => { }", TestOptions.RegularPreview,
                 // (1,1): error CS1073: Unexpected token '=>'
                 // -- [A] () => { }
-                Diagnostic(ErrorCode.ERR_UnexpectedToken, "-- [A] ()").WithArguments("=>").WithLocation(1, 1),
-                // (1,4): error CS1525: Invalid expression term '['
-                // -- [A] () => { }
-                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "[").WithArguments("[").WithLocation(1, 4));
+                Diagnostic(ErrorCode.ERR_UnexpectedToken, "-- [A] ()").WithArguments("=>").WithLocation(1, 1));
```

Comment:
>Diagnostic(ErrorCode.ERR_UnexpectedToken, "-- [A] ()").WithArguments("=>").WithLocation(1, 1)); [](http://example.com/codeflow?start=16&length=95)

It's interesting there is no error on `()`. Given that, consider adding the following expression tests to CollectionLiteralParsingTests.cs:
`[A, B]()`
`++[A, B]()`

Reply:
Will do.  In this case, tehse are just invocations.  Semantically that's meaningless.  But it's for binding to make that conclusion, not parsing.

Reply:
Done.